datastores-parent/datastore-tests/pom.xml

```diff
@@ -76,5 +76,10 @@
 			<artifactId>websocket-jakarta-server</artifactId>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<scope>compile</scope>
```

Comment:
the scope shouldn't be `compile`
It should be removed (so `test` will be used)

Reply:
`compile` scope is required at the moment. See the discussion in #1008